docs/reference/modules/cross-cluster-search.asciidoc

```diff
@@ -340,6 +341,15 @@ image:images/ccs/ccs-min-roundtrip-cluster-results.svg[]
 . The coordinating node sends a search request to each shard, including those in
 its own cluster. Each shard performs the search request independently.
 +
+[WARNING]
+====
+Cluster-level settings from the _local_ cluster are applied to each request.
```

Comment:
this is the section about not minimizing roundtrips right? I think "request" here may be confusing to users. The way I would try to explain this is that when you don't minimize roundtrips, the whole search is executed almost as if all the data was in a single cluster (the coordinating one, meaning that cluster where the coordinating node is located), hence the limits need to account for that. Is that what you meant?

Reply:
Thanks @javanna. That explanation was very helpful!

I've reworded the warning with 0b8c2619162fdb8789e5b40cd655d63501775885.